src/UAEventEmitter.ts

```diff
@@ -22,9 +27,12 @@ export class UAEventEmitter {
         }
       );
     }
-
-    this.eventEmitter.addListener('com.airship.pending_events', async () => {
-      return this.dispatchEvents(false);
+    this.eventEmitter.addListener('com.airship.ready', () => {
```

Comment:
I think you want to keep this `com.airship.pending_events`

Reply:
`com.airship.ready` gets called only when airship takes off, we want to dispatch events whenever a new pending event is available

Reply:
Oh yes, sorry I fixed that. That was copilot. My bad.